pom.xml

```diff
@@ -302,9 +302,18 @@
 
       <dependency>
         <groupId>org.codehaus.groovy</groupId>
-        <artifactId>groovy-all</artifactId>
-        <version>2.0.1</version>
-        <scope>test</scope>
+        <artifactId>groovy</artifactId>
```

Comment:
I switched to the smaller groovy jars so the extension jar isn't so large (shaved 6MB off that way)